src/controllers/login.js

```diff
@@ -0,0 +1,33 @@
+const bcrypt = require('bcryptjs');
```

Comment:
Work out with @ali-7 and see if you can make this file look any better, if you didn't understand what I mean by this or didn't know how to make a change then just ignore this comment, I only want to see what creative ideas you guys could give structuring your files, functions and functionalities.